Content.Client/Radiation/Systems/RadiationSystem.cs

```diff
@@ -0,0 +1,43 @@
+using Content.Client.Radiation.Overlays;
+using Content.Shared.Radiation.Events;
+using Robust.Client.Graphics;
+
+namespace Content.Client.Radiation.Systems;
+
+public sealed class RadiationSystem : EntitySystem
+{
+    [Dependency] private readonly IOverlayManager _overlayMan = default!;
+
+    public override void Initialize()
+    {
+        SubscribeNetworkEvent<OnRadiationOverlayToggledEvent>(OnOverlayToggled);
+        SubscribeNetworkEvent<OnRadiationOverlayUpdateEvent>(OnOverlayUpdate);
+        SubscribeNetworkEvent<OnRadiationOverlayResistanceUpdateEvent>(OnResistanceUpdate);
+    }
+
+    private void OnOverlayToggled(OnRadiationOverlayToggledEvent ev)
+    {
+        if (ev.IsEnabled)
+            _overlayMan.AddOverlay(new RadiationDebugOverlay());
+        else
+            _overlayMan.RemoveOverlay<RadiationDebugOverlay>();
+    }
```

Comment:
Does this get untoggled on shutdown? I don't think overlaymanager doesn't get touched on sim startup or shutdown so overlays need clearing manually.